frontend/src/components/common/form/Input.tsx

```diff
@@ -129,6 +132,7 @@ export const Input: React.FC<InputProps> = ({
           value={pattern ? restricted : rest.value ?? value}
           title={pattern ? restricted : rest.value ?? value}
           placeholder={placeholder}
+          aria-describedby={helpText ? `${field}-help-text` : undefined}
```

Comment:
improving component accessibility (ie screen readers) when help text is provided